setup.py

```diff
@@ -16,7 +16,7 @@
 with open(here / "README.md", encoding="utf-8") as f:
     readme = f.read()
 
-dev_requires = ["black", "pre-commit", "devtools", "jupyter", "flake8"]
+dev_requires = ["black", "pre-commit", "devtools", "jupyter", "flake8", "pyarrow"]
```

Comment:
Why is pyarrow a dev requirement and not a general requirement?

Reply:
[10:50 AM] Thom, Daniel
My reasons here:
Minimize the chance for package dependency conflicts. When these happen, they are a nightmare.Minimize our installation size for users.